source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,187 @@
+.. _install-configfile:
+
+Install using clr-installer and a configuration file
+####################################################
+
+This guide describes how to install |CL-attr| using the clr-installer tool
+with a configuration file. The configuration file (:file:`clr-installer.yaml`)
+can be used to duplicate the same installation configuration on more machines.
+
+
+System requirements
+*******************
+
+Ensure that your target system supports the installation:
+
+* :ref:`system-requirements`
+* :ref:`compatibility-check`
+
+Process
+*******
+
+This guide describes two methods for using a configuration file with the
+clr-installer tool. You can use either method to achieve the same goal. Choose
+the method that works best for your setup.
+
+If you are installing |CL| for the first time, we recommend Example 1.
+
+To clone an existing |CL| setup on another system, we recommend Example 2.
+
+Example 1
+=========
+
+This method uses an existing configuration file template to perform a new
+installation.
+
+Perform the following steps:
+
+#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+
+   For example:
+   https://download.clearlinux.org/releases/30010/clear/clear-30010-live-server.iso.xz
+
+#. Follow the instructions to :ref:`bootable-usb` based on your OS.
+
+#. Boot up the USB thumb drive.
+#. Select :guilabel:`Clear Linux OS` from the menu.
+#. In the console window, log in as root and set a password.
+#. Verify you have an IP address (network connection) and configure proxies (if needed).

```

Comment:
@MCamp859 - Pls change to "Verify you have a network connection to the Internet and configure proxy settings if you're working behind a firewall."

Reply:
Done here and in lines 169-170 also (same text).